src/server/status/index.js

```diff
@@ -1,11 +1,16 @@
 import ServerStatus from './server_status';
 import wrapAuthConfig from './wrap_auth_config';
+import { Metrics } from './metrics';
 
 export default function (kbnServer, server, config) {
   kbnServer.status = new ServerStatus(kbnServer.server);
 
   if (server.plugins['even-better']) {
-    kbnServer.mixin(require('./metrics').collectMetrics);
+    const metrics = new Metrics(config, server);
+
+    server.plugins['even-better'].monitor.on('ops', event => {
+      metrics.capture(event).then(data => { kbnServer.metrics = data; });
```

Comment:
mostly to provide a layer in between but i didn't have anything in particular in mind.   it could prevent modifications by reference or provide an event emitter.  we can come back to it later if it turns out its something we need.